Transform_liraries/QPolygonTransform_lib/unit_tests/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ add_custom_target(build-time-make-directory-qpoly ALL
   COMMAND ${CMAKE_COMMAND} -E make_directory ${OUTPUT_IMG_PATH}/test_library
   COMMAND ${CMAKE_COMMAND} -E make_directory ${OUTPUT_IMG_PATH}/test_color)
 
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../../Pictures/BITMAPA.png
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../../../Pictures/BITMAPA.png
```

Comment:
Proposal for future PR: please create special CMakeLists dedicated only to handle assets in the neighbor of assets. Relaing on relative path may be root cause of more issues.

Reply:
done